doc/UsersGuide/CustomizingTheWorkflow/ConfigWorkflow.rst

```diff
@@ -2107,8 +2107,9 @@ Non-default parameters for the Community Fire Behavior Model (CFBM) in SRW are s
    Height to interpolate winds to for calculating fire spread rate
 
 ``FIRE_ATM_FEEDBACK``: (Default: 0.0)
```

Comment:
Please update the default value for `FIRE_ATM_FEEDBACK` from `0.0` to `1.0`, since the default was changed in `ush/config_defaults.yaml` from `0.0` to `1.0`.

Reply:
Actually, seeing how you are setting `FIRE_ATM_FEEDBACK: 1.0` in both `tests/WE2E/test_configs/fire/config.UFS_FIRE_multifire_two-way-coupled.yaml` and `ush/config.fire.yaml`, you might want to change the default value of `FIRE_ATM_FEEDBACK` from `1.0` back to `0.0` in `ush/config_defaults.yaml` instead.

Reply:
@MichaelLueken by default we want users running the coupled version since that will give the most accurate results, so I would prefer the default be 1.0. I don't understand your reasoning for changing it back to 0.0.

Reply:
@mkavulich It's fine to keep the default set to `1.0`.  Keeping this set will require the update to the default in  `ConfigWorkflow.rst`, as well as setting `FIRE_ATM_FEEDBACK: 0.0` in the `UFS_FIRE_one-way-coupled` configuration file.  I figured that it would be easier to just change the default `FIRE_ATM_FEEDBACK: 1.0` to `0.0`, but keeping it as is and modifying two files to handle `1.0` is fine too.

Reply:
@MichaelLueken That setting is already included in `config.UFS_FIRE_one-way-coupled.yaml`, so no update is needed there. I have updated the incorrect default value in the docs.

Reply:
Thanks, @mkavulich!  I have approved the PR.